elastic/kibana
# [Fleet] Install a default ingest pipeline for datastreams without ones

## Description 

Related to #133740 

Install a default ingest pipeline for all the datastreams managed by Fleet event if there is none,
The ingest pipeline will be named like this `${dataStream.type}-${dataStream.dataset}-${packageVersion}` (Following the same convention as datastream ingest pipeline added by the package)

## How to test that PR? 

You can install a package and see that the datastreams without pipeline get one created without any processors 

<img width="1297" alt="Screen Shot 2022-06-15 at 10 07 29 AM" src="https://user-images.githubusercontent.com/1336873/173847897-f5ec8e86-1420-4ca2-992a-6683edd83b19.png">
<img width="1294" alt="Screen Shot 2022-06-15 at 10 07 48 AM" src="https://user-images.githubusercontent.com/1336873/173847909-c6faa164-1dd7-48e3-bd22-9aa8ee87d1a2.png">

I also tested to install all packages `node scripts/install_all_packages` and no errors did come out
